python/dgl/graphbolt/impl/temporal_neighbor_sampler.py

```diff
@@ -97,9 +97,15 @@ def sample_subgraphs(self, seeds, seeds_timestamp):
         num_layers = len(self.fanouts)
         # Enrich seeds with all node types.
         if isinstance(seeds, dict):
+            (dtype, device,) = (
+                list(seeds.values())[0].dtype,
```

Comment:
Instead of iterating over all values, it might better to only access the first element.